rclcpp/minimal_publisher/not_composable.cpp

```diff
@@ -30,12 +30,14 @@ int main(int argc, char * argv[])
   auto publisher = node->create_publisher<std_msgs::msg::String>("topic");
   auto message = std::make_shared<std_msgs::msg::String>();
   auto publish_count = 0;
+  rclcpp::WallRate loop_rate(2s);
```

Comment:
This is 0.5Hz, but before it was 2Hz, right?

Reply:
oops: e21d2d6f2414388235dd21ef7e4bbd907fc2bd3e puts it back to 2Hz